src/javascript/app/base/footer.js

```diff
@@ -13,10 +13,14 @@ const Footer = (() => {
             // logged in virtual clients with maltainvest financial landing company or
             // logged out clients with EU IP address
             if (Client.isLoggedIn()) {
-                showWarning((Client.get('landing_company_shortcode') === 'maltainvest' ||
+                const landing_company_shortcode = Client.get('landing_company_shortcode');
```

Comment:
@negar-binary @nazaninreihani don't we have `hide/show` logic already? 
https://github.com/binary-com/binary-static/blob/master/src/javascript/app/common/content_visibility.js

Reply:
we need to check for loggedin and logged out clients to exclude maltainvest, that does not have the logic I think